packages/agents/sequencer/src/lib/operations/execute/fastpath.ts

```diff
@@ -119,14 +119,20 @@ export const storeFastPathData = async (bid: Bid, _requestContext: RequestContex
     await channel.assertExchange(config.messageQueue.exchanges[0].name, config.messageQueue.exchanges[0].type, {
       durable: config.messageQueue.exchanges[0].durable,
     });
+    const queue = config.messageQueue.queues.find((it) => it.name == transfer.xparams!.originDomain);
+    await channel.prefetch(queue?.limit || 1);
```

Comment:
I don't think this is needed before the `publish`

Reply:
You are right, removed it.